frontend/src/assets/styles/themes/themeValues/colors.ts

```diff
@@ -1,9 +1,21 @@
 export type Colors = {
   primary: string;
+  secondary: string;
   regularText: string;
+  btnActivePrimary: string;
+  btnHoverPrimary: string;
+  btnActiveSecondary: string;
+  btnHoverSecondary: string;
```

Comment:
Super, załapałaś :)
Zmieniłbym nazewnictwo. W programowaniu nie powinniśmy używać skrótów, mimo że wydaje się to fajnym pomysłem.
Stosowanie skrótów bardzo zwiększa koszt utrzymania kodu.
W pracy u mnie występują skróty "tsf", "tsm" itp i bez wiedzy domenowej nigy nie poznamy co się za tym kryje.
Nawet w życiu, skrót "km" ma kilka rozwinięć. :)

Reply:
A mozę po prostu nazewać "primaryHover" bo czy to jest ściśle związane z przyciskiem czy też jeśli coś innego będzie miało hover to będzie miało już innny kolor?